apollo/src/graphql/model/modelQueries.ts

```diff
@@ -6,13 +6,18 @@ builder.queryFields((t) => ({
         type: [MLModel],
         args: {
             modelName: t.arg.string(),
+            limit: t.arg.int({ required: true }),
+            offset: t.arg.int({ required: true }),
```

Comment:
Could you add `defaultValue` to each of these and mark them as optional arguments? It might also make sense to convert `limit` into an enum for standard page sizes (maybe 10, 25, 50 or something like that)

Reply:
Yeah I could add default values.

I was going to specify the limits on the frontend, but I can do them here as well.

Reply:
Just given the variety of clients (frontend, python sdk, some madlad hitting the gql endpoint directly), it might make for a more consistent experience for things like that to be polite but firm suggestions maintained server-side

Reply:
Fun fact, trying to define an enum of integer values in graphql is really hard lol

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums

What's going wonky? Is Pothos just not playing nicely?

Reply:
Nvm I'm the greatest engineer to ever live.

Reply:
😂 the duality of man

Reply:
Okay, added a custom scalar type that takes in the values (10, 25, 50). If a request is made with a limit that is not null and is not within these values, an error is thrown.

Also created an array constant that holds the valid limits and hooked up a type to that so that they are always in sync.

![image](https://github.com/dstk-labs/dstk/assets/77359138/a5baa96d-1d13-49bd-81b6-108758dce1c1)

![image](https://github.com/dstk-labs/dstk/assets/77359138/99f22fda-2e58-456d-9476-ec3fe30e7d99)